tests/conftest.py

```diff
@@ -101,9 +101,10 @@ def get_keychain():
 class Mode(Enum):
```

Comment:
a bit of a nit, but shouldn't this be called `ConsensusMode`?

Reply:
That's indeed clearer, thanks. I sent #15794 if we're going with it.

Reply:
yeah, this interacts with Florin's chip-13 patch as well, so we might want to wait with renaming it until after his PR has landed too